jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/Resource.java

```diff
@@ -55,148 +46,22 @@
  * a file, a URL or an entry in a jar file.
  * </p>
  */
-public abstract class Resource implements ResourceFactory
+public abstract class Resource
```

Comment:
Can't this become an interface again?

Reply:
I've personally never seen `Resource` being an interface. I'd consider that only after all the other cleanups have been done, i.e.: that's a job for another PR.

Reply:
Hmm, I must be imaging things, I could have sworn that it briefly was an interface over the past couple of months, but I can find no proof of that.